### ISSUE TITLE
Rephrase the term 'Connection'

### ISSUE THREAD

**Issue by itagagaki** [1Remote/1Remote#768]
The term 'Connection' is used in several places, with different meanings, and they share a single term in xaml.
Settings > General > Connection means the behavior of the application when connecting or disconnecting.
'Connection' in almost protocol form means an account information.
But for serial communication, it is a description of the port.
When translating these into Japanese, it is difficult to find a term that fits all of them.
So please use separate terms instead of sharing the same terminology?

**Comment by VShawn** [comment]
It's ok to divide them into different words if needed.

> 'Connection' in protocol 

means connection info including host \ port \ com etc.

![image](https://github.com/user-attachments/assets/8792a496-e962-443e-a6c2-de7aedcd08c6)

> Settings > General > Connection

refers to all configuration options during the session connection process, including start, end and in the session.

If you are sure that different words are needed to express these two contexts in Japanese, then I will separate them. and feel free to inform me if any other term need a separation.

**Comment by itagagaki** [comment]
Yes, please. At least in Japanese, it is difficult to represent the two in the same word.

I think "connection" in each protocol setting should be "connection info" as you suggested. In Japanese it is "接続情報".

The options are... a bit trickier. I can't think of a simple term to describe "configuration options during the session connection process, including start, end, and in-session", but I'll let you know if I come up with a better term.
